doc/conf.py

```diff
@@ -46,16 +45,19 @@
     try:
         m = obj.__module__
         obj.__module__ = 'unittest' if m == 'unittest.case' else m
-    except Exception: pass
+    except Exception:
+        pass
 
 # This is a hack to prevent :ivar: docs from attempting to create a reference
 # Credit goes to https://stackoverflow.com/a/41184353/5096023
+
```

Comment:
Is this expected/intended?

Reply:
Seems like it treats a comment as a declaration. That's not really intended, but I'm not sure if that's really harmful either. That will make happy these kind of tools, so if we agree to proceed with that, diffs will be restricted to modified code in the future.